extensions/amp-sidebar/validator-amp-sidebar.protoascii

```diff
@@ -53,6 +55,16 @@ tags: {  # <amp-sidebar>
   amp_layout: {
     supported_layouts: NODISPLAY
   }
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-sidebar"
+}
+tags: { #<amp-sidebar> in amp-story
+  html_format: AMP
+  tag_name: "AMP-SIDEBAR"
+  spec_name: "amp-story >> amp-sidebar"
+  requires_extension: "amp-sidebar"
+  mandatory_ancestor: "AMP-STORY"
```

Comment:
should this be mandatory_parent? story is basically the new `body` tag, and sidebar is typically required to be nested immediately below `body`.

Reply:
You're right! Thanks for the catch.